model/office/callback.go

```diff
@@ -148,6 +155,7 @@ func saveFile(inst *instance.Instance, detector conflictDetector, downloadURL st
 	newfile.UpdatedAt = time.Now()
 	newfile.CozyMetadata.UpdatedAt = newfile.UpdatedAt
 	newfile.CozyMetadata.UploadedAt = &newfile.UpdatedAt
+	newfile.CozyMetadata.UploadedBy = &vfs.UploadedByEntry{Slug: OOSlug}
```

Comment:
@nono I think we have a problem here. From what I understood, the uploadedBy is only changed when the file is created. So after this route is called, its value always remains at `{"slug": "onlyoffice-server" }`.  For the front, it means that the modals are no longer displayed from this point. A more effective way would be to add an entry in updatedByApps and based the front on it. What do you think?

Reply:
In the same way, `CozyMetadata.UploadedAt` should not be modified either. We don't modify these two attributes when uploading a new version via Drive or Desktop.

Reply:
We do modify these two attributes when a file is uploaded (creation or content modification). It's the stack that does the job. IS it OK for you?

Reply:
I must have a bug in local, the stack does not modify the fields and it always remains in onlyoffice-server after a modification by OnlyOffice 🙁